components/filter/events.tsx

```diff
@@ -350,25 +359,41 @@ const Event = ({ visible, event, jsonLd }: EventProps) => {
             />
 
             <div>
-              {event.presenterName && (
+              {(event.presenterName || event.presenterList?.length > 0) && (
                 <EventDescItem
-                  label="Presenter"
-                  linkValue={event?.presenterProfileUrl}
-                  value={event.presenterName}
-                />
+                  label={
+                    event?.presenterName || event?.presenterList?.length === 1
+                      ? "Presenter"
+                      : "Presenters"
```

Comment:
do you think else condition will ever executed on given scenario?

Reply:
As per our conversation, the label should say "Presenter" so long as an overriding presenter name is provided.